### DISCUSSION TITLE
Hide the scrollbar from a Log widget while keeping the content scrollable

### DISCUSSION THREAD

**Discussion by superlevure** [Textualize/textual#3479] [Q&A] [answered]
Is there a way to hide the scrollbar from a `Log` widget while keeping its content scrollable? 

I've tried:

- `overflow-y: hidden;` but this disables the ability to scroll
- `scrollbar-size-vertical: 0;` but 0 is not allowed
- `log.show_vertical_scrollbar = False` but it does nothing
- `log._show_vertical_scrollbar = False` same

Thank you for your help!

**Comment by davep** [comment, marked as answer]
Unfortunately, at the moment, there isn't really an easy way to remove a scrollbar and retain scrolling. We think it does make sense that you should be allowed to set the size to `0` though so I'll add an issue for that; seems like a valid requirement.

**Reply by superlevure** [reply]
Thanks  for the quick answer!

**Reply by davep** [reply]
Raised #3482.
